contracts/plugins/assets/morpho-aave/MorphoNonFiatCollateral.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.19;
+
+import { CollateralConfig, MorphoFiatCollateral } from "./MorphoFiatCollateral.sol";
+import { FixLib, CEIL } from "../../../libraries/Fixed.sol";
+import { OracleLib } from "../OracleLib.sol";
+// solhint-disable-next-line max-line-length
+import { AggregatorV3Interface } from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+/**
+ * @title MorphoNonFiatCollateral
+ * @notice Collateral plugin for a Morpho pool with non-fiat collateral, like WBTC
+ * Expected: {tok} != {ref}, {ref} is pegged to {target} unless defaulting, {target} != {UoA}
+ */
+contract MorphoNonFiatCollateral is MorphoFiatCollateral {
+    using OracleLib for AggregatorV3Interface;
+    using FixLib for uint192;
+
+    AggregatorV3Interface public immutable targetUnitChainlinkFeed; // {UoA/target}
```

Comment:
comment should be `{target/ref}`, I think, assuming this feed should be used for the `pegPrice`